server/gql/models/studentSubjects/index.js

```diff
@@ -1,61 +1,12 @@
-import { getNode } from '@gql/node';
-import db from '@database/models';
-import { totalConnectionFields } from '@utils/index';
-import { getQueryFields, TYPE_ATTRIBUTES } from '@server/utils/gqlFieldUtils';
-import { GraphQLID, GraphQLInt, GraphQLNonNull, GraphQLObjectType } from 'graphql';
-import { createConnection } from 'graphql-sequelize';
-import { timestamps } from '../timestamps';
-import { sequelizedWhere } from '@server/database/dbUtils';
+import { studentSubject, studentSubjectQuery, StudentSubjectConnection } from './query';
+import { studentSubjectMutation } from './mutation';
```

Comment:
this doesn't adhere to what we spoke about. So we would like to create the following files for each gql model. 

1. index.js (this contains information about the fields in the model
2. query.js this contains setup for the singular query
3. list.js this contains the setup for the connection
4. mutation this contains the setup for the mutations
5. in case there are custom mutations they will go in their own files as follows 
   - customCreateMutation.js
   - customUpdateMutation.js
   - customDeleteMutation.js

Reply:
Okay.